.gitignore

```diff
@@ -2,7 +2,6 @@
 Gemfile.lock
 .*.sw?
 pkg
-spec/fixtures
```

Comment:
I actually think I removed that only for temporary addition of the modules link.  That can likely be added back in.